Reflective Report.tex

```diff
@@ -0,0 +1,85 @@
+% Please do not change the document class
+\documentclass{scrartcl}
+
+% Please do not change these packages
+\usepackage[hidelinks]{hyperref}
+\usepackage[none]{hyphenat}
+\usepackage{setspace}
+\usepackage{graphicx}
+\doublespace
+
+% You may add additional packages here
+\usepackage{amsmath}
+
+% Please include a clear, concise, and descriptive title
+\title{Reflective Essay}
+
+% Please do not change the subtitle
+\subtitle{COMP120 - Reflective Report}
+
+% Please put your student number in the author field
+\author{1503048}
+
+\begin{document}
+
+\maketitle
+
+\abstract{In this reflective report, I'll be reviewing my learning while reflecting on my projects and approaches for the first semester.}
+
+\section{Introduction}
+
+The three weakness I'll be exploring in this report are: time management, critical evaluation, . All of these skills are significant to me for professional practise and have been emphasised during my first semester.
```

Comment:
An active voice is preferred. For example, "the three weaknesses I explore in this report".